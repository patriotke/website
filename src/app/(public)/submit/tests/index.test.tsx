/* eslint-disable import/no-extraneous-dependencies */
import '@testing-library/jest-dom';
// eslint-disable-next-line @typescript-eslint/no-unused-vars
import { render, screen } from '@testing-library/react';
import Submit from '../page';

describe('Submit', () => {
  it('renders Submit', () => {
    render(<Submit />);

    const { container } = render(<Submit />);
    expect(container).toMatchSnapshot();
  });
});
