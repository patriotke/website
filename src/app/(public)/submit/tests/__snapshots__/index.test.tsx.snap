// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Submit renders Submit 1`] = `
<div>
  <div
    class="container mx-auto"
  >
    <div>
      SubmissionToolbar
    </div>
  </div>
</div>
`;
